Product/SoapUI_Test/RegressionSuite/Passthru/AuditLogging-Passthrough/COREX12BatchSubmitRequest.properties

```diff
@@ -51,12 +51,12 @@ ActiveParticipant_Dest.RoleIDCode.@code=110152
 ActiveParticipant_Dest.RoleIDCode.@codeSystemName=DCM
 ActiveParticipant_Dest.RoleIDCode.@displayName=Destination
 
-AuditSourceIdentification.@AuditEnterpriseSiteID=Gateway 1
+AuditSourceIdentification.@AuditEnterpriseSiteID=DoD
```

Comment:
This property, among many others, is repeated across nearly 20 property files.  Is there any reason we don't want this just once at the project level?